client/src/components/DisplayTable/DisplayTable.js

```diff
@@ -84,35 +54,73 @@ export class DisplayTable extends React.Component {
   constructor(props) {
     super(props);
 
-    this.state = get_default_state_from_props(props);
-  }
-  static getDerivedStateFromProps(nextProps, prevState) {
```

Comment:
Looks like this code is from Stephen on covid branch. @Stephen-ONeil can these go away like this?

Reply:
Yeah, this change was important. If a component derives a bunch of state from props in the constructor only, then it's not going to respond to prop changes. There's some case added in the covid panels where `DisplayTable` props can be changed post-mount, so this regression breaks those at least.

Reply:
I was mixing this up with the `derived_menu_state` from the `Typeahead` 😂🤦‍♂️

Thanks for pointing it out!